plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -65,6 +66,7 @@
           group="non-appengine-standard"
           name="Objectify"
           tooltip="%objectify.tooltip"
+          javaVersion="1.8"
```

Comment:
why was this needed?

Reply:
While looking at why the test failed, I noticed the`Library` instance for Objectify had`javaVersion` set to `"1.7"`. Objectify 6 requires Java 8.